src/gtimelog/data/org.gtimelog.gschema.xml

```diff
@@ -8,6 +8,13 @@
         <value nick="summary" value="3"/>
     </enum>
 
+    <enum id="org.gtimelog.LogOrder">
+        <value nick="asis" value="1"/>
```

Comment:
I'd be slightly happier with `as_is` or `as-is`.

Or we could be explicit and go with `chronological`?

Or I could stop nitpicking and learn to live with `asis`, which I help but pronounce as a single word.

Reply:
I like it.